components/Pagination/sub-components/Desktop.jsx

```diff
@@ -46,6 +47,7 @@ const Desktop = ({
           active={activePage === page}
           onClick={handlePageClick(page)}
           href={handleHref(page)}
+          rel={page > 1 ? 'nofollow' : ''}
```

Comment:
This must not be the standard behavior (at least there must be a flag to activate this), because this is useful only to specific cases.

Reply:
@jgpacker @alisson-suzigan told me that this is a new guideline of our SEO here at Catho. Based on that I've created this way.

Reply:
This is a definition our specific case, and not for other users. There must be a flag to activate this (e.g. `onlyFollowFirstPage`)

Reply:
The flag was added.